docs/faq.qmd

```diff
@@ -33,6 +43,18 @@ Yes, Bambi supports inference on GPUs and TPUs using the numpyro and blackjax ba
 See the API for "fit" method for more details 
 [here](https://bambinos.github.io/bambi/api/Model.html#bambi.Model.fit).
 
+### My sampler through errors/indicating divergences, what should I do?
```

Comment:
I am not sure this deserves an FAQ. This is a more general question regarding sampling. Not necessarily specific to Bambi.